ui/component/viewers/videoViewer/internal/plugins/videojs-hls-quality-selector/plugin.js

```diff
@@ -238,35 +287,36 @@ class HlsQualitySelectorPlugin {
    * @param {number} height - A number representing HLS playlist.
    */
   setQuality(height) {
+    const { doSetDefaultVideoQuality } = this.config;
     const qualityList = this.player.qualityLevels();
 
     // Set quality on plugin
     this._currentQuality = height;
+    doSetDefaultVideoQuality(height);
```

Comment:
This is changing the default every time I make a change from the player.

If we want the behavior of "carry forward the latest selection as the new default", then the selector in Settings Page no longer make sense and can be removed.

I think the default should only be changed from the Settings Page, and any changes from the player will just be a transient setting for that specific video. I think this model is simpler (both in code and behavior).